sycl/include/CL/sycl/INTEL/esimd/esimd_view.hpp

```diff
@@ -44,31 +47,17 @@ template <typename BaseTy, typename RegionTy> class simd_view {
   /// type of the base object type.
   using element_type = typename ShapeTy::element_type;
 
-  // TODO @rolandschulz
-  // {quote}
-  // Why is this and the next constructor public ? Those should only be called
-  // internally by e.g.select, correct ?
-  // {/quote}
-  //
   /// @{
   /// Constructors.
+
+private:
   simd_view(BaseTy &Base, RegionTy Region) : M_base(Base), M_region(Region) {}
   simd_view(BaseTy &&Base, RegionTy Region) : M_base(Base), M_region(Region) {}
 
-  // TODO @rolandschulz
-  // {quote}
-  // Is this intentional not a correct copy constructor (would need to be const
-  // for that)? I believe we agreed that simd_view would have a deleted copy and
-  // move constructor.Why are they suddenly back ?
-  // {/quote}
-  // TODO @kbobrovs
-  // copy constructor is still incorrect (no 'const'), move constructor is still
-  // present.
-  //
-  // Disallow copy constructor for simd_view.
-  simd_view(simd_view &Other) = delete;
-  simd_view(simd_view &&Other)
-      : M_base(Other.M_base), M_region(Other.M_region) {}
+public:
```

Comment:
Side note: access specifiers of deleted constructors is pretty meaningless here. This 'public' doesn't actually do anything as far as I remember.

Reply:
this is just restoring the visibility for the following members (conversion, assignments,...) after injected `private` for the two constructors.